### DISCUSSION TITLE
Emacs

### DISCUSSION THREAD

**Discussion by GalGavu** [dustinlyons/nixos-config#61] [Q&A]
Hi,
Is there any way to simply not add emacs? :) I'm not using emacs at all and don't need it but it is in so many places so I was wondering if there might be just a simple switch to turn it off?

**Comment by dustinlyons** [comment]
Hi,

One day I'd like to make this part of the installer. Until then, here's what you do:

**NixOS**
Disable Emacs here: https://github.com/dustinlyons/nixos-config/blob/68ebaeadcf60b75a58111ebbc59558f86ce3d9f0/hosts/nixos/default.nix#L238

**Darwin**
Remove from system packages here: https://github.com/dustinlyons/nixos-config/blob/68ebaeadcf60b75a58111ebbc59558f86ce3d9f0/hosts/darwin/default.nix#L40

**Comment by GalGavu** [comment]
Thanks
